### PR TITLE
Updating cmake and adding msvc specifics

### PR DESCRIPTION
**Pull request by aescande**

### CODE REVIEW

**Comment by aescande** [comment]
Updated to the current master and add some more fixes for tests in Visual Studio and python bindings.

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ cmake_minimum_required(VERSION 2.8)
 include(cmake/base.cmake)
 include(cmake/boost.cmake)
 include(cmake/eigen.cmake)
+INCLUDE(cmake/msvc-specific.cmake)
```
Comment (gergondet) [inline CMakeLists.txt]: `INCLUDE` -> `include`

File: CMakeLists.txt [hunk 2]
```diff
@@ -70,6 +72,16 @@ endif()
 
 add_definitions("-DEIGEN_QLD_EXPORT")
 
+# For MSVC, set local environment variable to enable finding the built dll
+# of the main library when launching ctest with RUN_TESTS
+if(MSVC)
+  if(USE_FORTRAN_SUBDIRECTORY_DEFAULT)
```
Comment (gergondet) [inline CMakeLists.txt]: Shouldn't this be `USE_FORTRAN_SUBDIRECTORY`?

File: CMakeLists.txt (hunk 1 above)
Comment (aescande) [reply CMakeLists.txt]: fixed

File: CMakeLists.txt (hunk 2 above)
Comment (aescande) [reply CMakeLists.txt]: Shouldn't change anything in most case, but you're right. Fixed.

**Pull request merged**